packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.routes.js

```diff
@@ -1,26 +1,157 @@
-import controller from './billing-main-history.controller';
-import template from './billing-main-history.html';
-
-angular.module('Billing').config(($stateProvider) => {
-  $stateProvider.state('app.account.billing.main.history', {
-    url: '/history?filters',
-    params: {
-      filters: {
-        value: '[]',
-        squash: true,
+import { Bill, DebtAccount } from '@ovh-ux/manager-models';
+import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
+import get from 'lodash/get';
+import omit from 'lodash/omit';
+import partition from 'lodash/partition';
+
+const mapDateFilter = ({ comparator, reference }) => {
```

Comment:
why do you have "reference" that seems to be an array as function parameter but only use the first item ?
maybe you just need this item as function parameter no?

Reply:
The parameter here is an object that was destructured but I can change to pass several parameters instead